.github/workflows/macos.yaml

```diff
@@ -0,0 +1,203 @@
+name: macOS
+
+# To debug a job, add the following step:
+# - name: Debug with tmate
+#   uses: mxschmitt/action-tmate@v3
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    paths-ignore:
+      - '.github/workflows/linux.yaml'
+
+env:
+  COZY_DESKTOP_DIR: "${{ github.workspace }}/cozy-desktop"
+  COZY_STACK_STORAGE: "${{ github.workspace }}/storage"
+  COZY_URL: "http://localhost:8080"
+  COZY_PASSPHRASE: "cozy"
+  GOPROXY: "https://proxy.golang.org"
+  GO111MODULE: "on"
+  NO_BREAKPOINTS: ${{ 1 }}
+  NODE_ENV: "test"
+  NPM_CONFIG_PROGRESS: ${{ false }}
```

Comment:
You have `NO_BREAKPOINTS` and `NPM_CONFIG_PROGRESS` on macOS but not on Linux. Is it expected?

Reply:
It is expected for `NO_BREAKPOINTS` since it's related to Chokidar scenarios. It's not for the other variable but I should completely remove it as we use `yarn` instead of `npm`.